scripts/get_graphids.py

```diff
@@ -0,0 +1,160 @@
+#! /usr/env python3
+
+import argparse
+import aiohttp
+import asyncio
+import async_timeout
+import posixpath
+from taskcluster.async import Index
+from taskcluster.async import Queue
+
+
+HG_URL = 'https://hg.mozilla.org/'
+
+REPOS = ('releases/mozilla-beta', 'releases/mozilla-release',
+         'releases/mozilla-esr52', 'mozilla-central')
+
+""" http://mozilla-version-control-tools.readthedocs.io/en/latest/hgmo/pushlog.html?highlight=json-pushes#json-pushes-command """  # noqa
+JSON_PUSHES = 'json-pushes?changeset={rev}'
+
+ACTION_INDEX = 'gecko.v2.{repo}.pushlog-id.{pushid}.actions'
```

Comment:
We could also add a new revision index for actions if that helps.

Reply:
It probably would help, but wouldn't necessarily help the trains on beta/release now. So wouldn't eliminate this code entirely